src/views/error.ejs

```diff
@@ -0,0 +1,3 @@
+<h1><%= message %></h1>
```

Comment:
What templating language is this? Why this one?

Reply:
Not a great answer, but it's the one I am familiar with. It was quick to learn as you write regular HTML and wrap your JS with <% %>.